test-case/check-pause-release-suspend-resume.sh

```diff
@@ -145,8 +150,15 @@ expect {
             exit 1
         }
 
+        #get the type of platform
+        set retval [catch { exec uname -a | grep -o glk } platform]
+
```

Comment:
Get platform from `uname` is a bad, unreliable workaround. What you see in our CI device is just from its hostname following our naming convention, so it's unportable and unreliable.